jenkins_jobs/registry.py

```diff
@@ -19,10 +19,9 @@
 import operator
 import pkg_resources
 import re
-import yaml
 
 from jenkins_jobs.errors import JenkinsJobsException
-from jenkins_jobs.formatter import CustomFormatter
+from jenkins_jobs.formatter import deep_format
```

Comment:
I didn't change that? I checked that my branch was in synch with 'experimental' i don't understand why this is here